### DISCUSSION TITLE
file manager

### DISCUSSION THREAD

**Discussion by linkoog** [podman-desktop/podman-desktop#5509] [Ideas]
I hope，podman-desktop   take  file manager  to container manager。when click into  container，I can only use SSH to modify files

**Comment by benoitf** [comment]
@linkoog you want to be able to edit files from your container within Podman Desktop ?

**Reply by vmcbaptista** [reply]
Ideally yes, but starting by being able to navigate through to the file system and open the files would already be very useful.
Docker desktop has this feature https://docs.docker.com/desktop/use-desktop/container/#files
![fe2.png](https://github.com/user-attachments/assets/16eb1f2e-3226-4a3d-832f-4cb8539015aa)

VS Code docker extension also allow us to navigate through the files of running containers.

**Comment by ramarro123** [comment]
@benoitf i think yes, and it's a very convenient feature. In docker desktop there is the option to open a "lightweight" filemanager to browse files in the running container and do some basic edit.
